src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -97,6 +98,8 @@ public String toSingularFormString() {
             return "team";
         case OWN_TEAM:
             return "team";
+        case CUSTOM:
+            return "custom feedback participant";
```

Comment:
What are the places that this will be shown, perhaps we should change the whole message rather than just use "custom feedback participant"